Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -340,6 +344,15 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
         target.CSSAgent.getInlineStylesForNode.invoke({nodeId: this._node.id}, wrap.call(this, fetchedInlineStyles, fetchedInlineStylesPromise));
         target.CSSAgent.getComputedStyleForNode.invoke({nodeId: this._node.id}, wrap.call(this, fetchedComputedStyle, fetchedComputedStylesPromise));
 
+
+        if (InspectorBackend.hasCommand("CSS.getCustomFonts")) {
+            target.CSSAgent.getCustomFonts((error, fontFamilyNames) =>{
```

Comment:
calling this every time we create/update a `WI.DOMNodeStyles` seems like way too much since it gets all fonts for the entire page, not just for a specific node (which is really what `WI.DOMNodeStyles` is about

id suggest moving this to `WI.CSSManager` and having an event from the backend (e.g. `CSS.fontsChanged`) that then triggers this to be called again, as that way we dont call it more than once for the same data

Style: missing space `=> {`